### PR TITLE
refactor(FX-4424): cleanup cascading auction flags

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [] 

### Description

- cleanup cascading flags

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- cleanup cascading auction flags - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Components/Bidding/Screens/ConfirmBid.tests.tsx [hunk 1]
```diff
@@ -869,17 +865,6 @@ describe("cascading end times", () => {
       expect(timerText).toBeTruthy()
     })
   })
-
-  describe("with cacsading end time feature disabled", () => {
```
Comment (dimatretyak) [inline src/app/Components/Bidding/Screens/ConfirmBid.tests.tsx:873]: **suggestion**: remove `describe("with cacsading end time feature enabled", () => {...})`

File: src/app/Scenes/Artwork/Components/CommercialInformation.tests.tsx [hunk 2]
```diff
@@ -333,10 +332,6 @@ describe("CommercialInformation buttons and coundtown timer", () => {
   })
 
   describe("when the enable cascading end time feature is on", () => {
```
Comment (dimatretyak) [inline src/app/Scenes/Artwork/Components/CommercialInformation.tests.tsx:334]: **suggestion**: remove `describe("when the enable cascading end time feature is on", () => {...}`

File: src/app/Scenes/Home/Components/SalesRail.tests.tsx [hunk 3]
```diff
@@ -244,9 +244,6 @@ describe("image handling when AREnableArtworksConnectionForAuction is enabled",
 
 describe("SalesRail Subtitle", () => {
   describe("with cascading feature flag switched ON", () => {
```
Comment (dimatretyak) [inline src/app/Scenes/Home/Components/SalesRail.tests.tsx:246]: **suggestion**: remove `describe("with cascading feature flag switched ON", () => {...})`

File: src/app/Scenes/Sale/SaleHeader.tests.tsx [hunk 4]
```diff
@@ -66,37 +63,7 @@ describe("SaleHeader", () => {
...
-        }),
-      })
-    )
-
-    expect(extractText(tree.root.findByProps({ testID: "sale-header-hero" }))).toBe(
-      "Auction closed"
-    )
-  })
-
   describe("when the cascading end time feature flag is on", () => {
```
Comment (dimatretyak) [inline src/app/Scenes/Sale/SaleHeader.tests.tsx:66]: **suggestion**: remove `describe("when the cascading end time feature flag is on", () => {...})`

File: src/app/Scenes/Sales/Components/SaleListItem.tests.tsx [hunk 5]
```diff
@@ -14,11 +12,6 @@ describe("SaleListItem", () => {
 
   describe("SalesRail Subtitle", () => {
     describe("with cascading feature flag switched ON", () => {
```
Comment (dimatretyak) [inline src/app/Scenes/Sales/Components/SaleListItem.tests.tsx:14]: **suggestion**: remove `describe("with cascading feature flag switched ON", () => {...})`

**Review by dimatretyak — COMMENTED** [review]
LGTM. Left some comments/suggestions

**Comment by gkartalis** [comment]
Thanks @dimatretyak! Will commit all of the suggestions after my meetings

**Review by dimatretyak — APPROVED** [review]
LGTM 👍

**Pull request merged**